Formula/mariadb@10.1.rb

```diff
@@ -24,7 +24,6 @@ class MariadbAT101 < Formula
   deprecated_option "with-tests" => "with-test"
 
   depends_on "cmake" => :build
-  depends_on "pidof" unless MacOS.version >= :mountain_lion
```

Comment:
Sharp eyes 👍 
Anyway it was a leftover, `pidof` is no longer used. And if I read https://github.com/Homebrew/homebrew-core/commit/7c6dc47bf261616e75542d8ff10ec1326e31ffe3#diff-16c08a700c21562997855c92d2787d15L118 correctly for some time already.